02.calendar/calendar.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+options = OptionParser.new
+
+params = {}
+options.on('-m month') {|v| v }
+options.on('-y year') {|v| v }
```

Comment:
オプションが指定されたときに何もしないのであればブロックは不要ですね。
```suggestion
options.on('-m month')
options.on('-y year')
```